### DISCUSSION TITLE
Mythic/Heroic Maghteridon Lair

### DISCUSSION THREAD

**Discussion by ta55el** [chromiecraft/chromiecraft#5954] [Suggestions]
This is my suggestion about ML Mythic mode. If would you like to use it i can make it for every boss if u'd like to.

The fight: Magtheridon has 5 adds that must be killed. He is banished for the first two minutes of the fight, during which you only have to deal with the adds. After two minutes he becomes active.

Hellfire Channelers:

Melee hits for ~1000 dmg on tank before buffs
Each channeler has roughly 600,000 hitpoints.
Shadow Volley hits all targets every 15 sec within 40 yards for roughly 2000 damage and cant be interrupted
Dark Mending: Heals a Hellfire Channeler for 200,000 HP. 2 second cast time. Cast at 75, 50, 25% HP.
Summon Abyssal: Summons an infernal. Each channeler summon 1 infernal at 80% HP. Infernals last until Maghteridon's dead.
After you kill 1 Channeler, the other 4 gain a buff (Soul Transfer) which increases their damage and casting speed by 30%, and size by 20%. Stacks as you kill Hellfire Channelers.

Burning Abyssal:

10M HP
melee hits for ~500 dmg on tank
Fire Aura - 10y range, 500 fire dmg every 3 sec
Abyssal will deal 2625-3375 with a 10 yard range to the player on which it lands.
Fireblast: 3300 damage on random player, 20 yard range every 20 sec.
Abyssals cant be CCd.

Magtheridon:

14M HP
Basic melee hits for 6-7k on a well-geared tank.
Quake: A knockback that hits every player every second for 7 seconds. First cast is 40 secs after Magtheridon activate. Interrupts spells and then is cast every 60 secs.
Blast Nova: Deals roughly 3000 damage every 2 seconds for 10 seconds to every player in the lair. First cast is 1 mins after Magtheridon activate and then cast every 60 sec.
Cleave: Standard cleave, can hit 8k+ on tank.
Conflagration: 6k fire damage over 8 sec

Manticron Cubes:

Will deal 800 damage per second to a player channeling it.
Applies Mind Exhaustion to the player, preventing him from clicking it again for another 5 minutes. Magtheridon takes 300% damage while all 5 cubes are active.

At 30% he will shatter the walls of his lair, causing the roof to crash down on the entire raid dealing 5250-6750 physical damage to every member of the raid.
For the remaining of the fight the ceiling will randomly every 5 sec cave in on players, dealing 87500-112500 damage to players standing underneath the cave in (8 yards radius).

Also any valuable improvements or suggestions are always welcome from you ;)

**Comment by balleny** [comment]
I mean practically, if you want to make the fight more challenging, without changing the mechanics much, i see some options:
(these are examples, combining wrongly may result in the boss not being killable)

### Phase 0
- increased HP
- when an add dies, the buff it grants to the others is granted twice
- instead of casting heal when one add reaches 60% or 25%, trigger them casting heal every 25%
- make shadowbolt volley not interruptable (but then ensure it is cast only by one at a time)
- reduce the Infernal spawn frequency but at the same time they dont despawn automatically any longer
- higher infernal spawn rate
- more damage from infernals fire aura

### Phase 1
- higher frequency of fires
- cubes being fixed that you cannot click them with the debuff
- cube channeling causing damage to the channeler
- cube debuff lasting 2 mins (or whatever is required for 20 clickers)

### Phase 1,5 (30%)
- Cubes dont work any longer, you have to heal the raid through the blast nova damage
- more debris

**Comment by QuantaCosta** [comment]
I suggest to scrap the idea of mythics or scalings alltogether and focus on getting the "normal" stuff working as smooth as possible.
It's already hella weird that if something breaks with regards of raiding, it appears to roll back to a state similar to PTR.
On top of that, there are tickets open for - in my honest opinion- critical bugs which are way too long in the pipeline to realistically ever get properly fixed.
Oh and, don't let me start talking about the "inhouse made" tank shortage.
Everyone who leveled a tank and experienced the transition from normal tbc dungeons to heroics knows what i'm hinting at ;)

**Comment by Nyeriah** [comment]
Rejecting this, as the custom mechanics idea was scrapped away.
